modules/effects/src/effect_notification.ts

```diff
@@ -31,7 +31,12 @@ export function reportInvalidActions(
 }
 
 function isAction(action: any): action is Action {
-  return action && action.type && typeof action.type === 'string';
+  return (
+    typeof action !== 'function' &&
+    action &&
+    action.type &&
+    typeof action.type === 'string'
+  );
```

Comment:
I think you don't need to wrap the return in brackets. This could be reverted to what it was before.

Reply:
Oh, sorry, I didn't realize that! I had already wondered why prettier always jumped on when I committed something on NgRx. That doesn't happen with my private repos.  I thought it was something like a hooks coming from ngrx repo.